doc/languages-frameworks/javascript.section.md

```diff
@@ -275,6 +275,51 @@ mkYarnPackage rec {
   - The `echo 9` steps comes from this answer: <https://stackoverflow.com/a/49139496>
   - Exporting the headers in `npm_config_nodedir` comes from this issue: <https://github.com/nodejs/node-gyp/issues/1191#issuecomment-301243919>
 
+### buildNpmPackage {#javascript-buildNpmPackage}
+
+`buildNpmPackage` allows you to package npm-based projects in Nixpkgs without the use of an auto-generated dependencies file ([node2nix](#javascript-node2nix)):
```

Comment:
> This introduction needs at least a note on how this works in principle, and that it takes a superset the parameters used for `stdenv.mkDerivation` (and link to that). Otherwise your listing of parameters will be incomplete. We have the [exact same problem in the Nix manual](https://github.com/NixOS/nix/issues/6817#issuecomment-1269431748), and it's really confusing to readers.

Do you still want me to specify this?